### PR TITLE
Implement benchmarking using Levenshtein distance.

### PR DESCRIPTION
**Pull request by martinzlocha**
Blocked by #11, will rewrite to use logging when merged.

### CODE REVIEW

File: WLC/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+def run_benchmarks():
+    print('=== Whiteboard Live Coding Benchmarking ===')
+    print('Uses Levenshtein distance to calculate the difference and then uses that to calculate accuracy.')
+    print()
+
+    for i in range(1, 6):
+        benchmark_file('assets/examples/images/example_{}.png'.format(i))
+
+
+run_benchmarks()
```
Comment (PaulSonOfLars) [inline WLC/benchmark.py]: no newline at EOF
Also, add an  `if __name__ == '__main__': ` around the run_benchmarks() call.

File: WLC/benchmark.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from WLC.image_processing.camera import Camera
+from WLC.image_processing.preprocessor import Preprocessor
+
+import editdistance
+
+
+def _get_expected_code(file_name):
+    file_name = file_name.replace('images', 'annotations')
+    file_name = re.sub(r"\..*", ".txt", file_name)
+    file_name = Camera().get_full_path(file_name)
```
Comment (PaulSonOfLars) [inline WLC/benchmark.py:12]: This seems very fragile, a rename would break the replace

Comment (martinzlocha) [reply WLC/benchmark.py:12]: Yeah but not sure thats the better solution. We need some standard.

File: WLC/benchmark.py (hunk 1 above)
Comment (martinzlocha) [reply WLC/benchmark.py]: Fixed

**Review by PaulSonOfLars — APPROVED** [review]

**Pull request merged**